src/python/pants/option/global_options.py

```diff
@@ -63,95 +66,114 @@ class ExecutionOptions:
             process_execution_remote_parallelism=bootstrap_options.process_execution_remote_parallelism,
             process_execution_cleanup_local_dirs=bootstrap_options.process_execution_cleanup_local_dirs,
             process_execution_use_local_cache=bootstrap_options.process_execution_use_local_cache,
             process_execution_cache_namespace=bootstrap_options.process_execution_cache_namespace,
-            remote_instance_name=bootstrap_options.remote_instance_name,
-            remote_ca_certs_path=bootstrap_options.remote_ca_certs_path,
-            remote_oauth_bearer_token_path=bootstrap_options.remote_oauth_bearer_token_path,
-            remote_execution_extra_platform_properties=bootstrap_options.remote_execution_extra_platform_properties,
-            remote_execution_headers=bootstrap_options.remote_execution_headers,
-            remote_execution_overall_deadline_secs=bootstrap_options.remote_execution_overall_deadline_secs,
             process_execution_local_enable_nailgun=bootstrap_options.process_execution_local_enable_nailgun,
-            remote_cache_read=bootstrap_options.remote_cache_read,
-            remote_cache_write=bootstrap_options.remote_cache_write,
-            remote_cache_eager_fetch=bootstrap_options.remote_cache_eager_fetch,
+            # Remote store setup.
+            remote_store_server=bootstrap_options.remote_store_server,
+            remote_store_thread_count=bootstrap_options.remote_store_thread_count,
+            remote_store_chunk_bytes=bootstrap_options.remote_store_chunk_bytes,
+            remote_store_chunk_upload_timeout_seconds=bootstrap_options.remote_store_chunk_upload_timeout_seconds,
+            remote_store_rpc_retries=bootstrap_options.remote_store_rpc_retries,
+            remote_store_connection_limit=bootstrap_options.remote_store_connection_limit,
             remote_store_initial_timeout=bootstrap_options.remote_store_initial_timeout,
             remote_store_timeout_multiplier=bootstrap_options.remote_store_timeout_multiplier,
             remote_store_maximum_timeout=bootstrap_options.remote_store_maximum_timeout,
+            # Remote cache setup.
+            remote_cache_eager_fetch=bootstrap_options.remote_cache_eager_fetch,
+            # Remote execution setup.
+            remote_execution_server=bootstrap_options.remote_execution_server,
+            remote_execution_extra_platform_properties=bootstrap_options.remote_execution_extra_platform_properties,
+            remote_execution_headers=bootstrap_options.remote_execution_headers,
+            remote_execution_overall_deadline_secs=bootstrap_options.remote_execution_overall_deadline_secs,
         )
 
 
-_CPU_COUNT = len(os.sched_getaffinity(0)) if hasattr(os, "sched_getaffinity") else os.cpu_count()
+_CPU_COUNT = (
+    len(os.sched_getaffinity(0)) if hasattr(os, "sched_getaffinity") else os.cpu_count()
+) or 2
```

Comment:
NB: this could have been `None` before and caused a runtime type exception. MyPy warned us of this. 2 seemed like a sensible fallback.

Reply:
When we upgrade to Tokio 1.0, the need for this will probably go away since Tokio can figure out the number of cores itself.

Reply:
We need to define this in Python so that we can have a default for the options system.

Reply:
To clarify, I meant the need for `_CPU_COUNT` entirely will probably go away with the Tokio 1.0 upgrade.

Reply:
(and any options using it but will consider that once Tokio 1.0 upgrade is unblocked)